ui-tests/src/test/java/io/syndesis/qe/pages/customizations/connectors/wizard/steps/UploadSwaggerSpecification.java

```diff
@@ -59,7 +59,9 @@ public void upload(String source, String url) {
     }
 
     public void uploadFileFromPath(String path) {
-        $(Input.UPLOAD_A_OPENAPI_FILE).shouldBe(visible).click();
+        // this radio element is always "checked" - due to this bug I have to click two times :(
+        $(Input.UPLOAD_AN_OPENAPI_FILE).shouldBe(visible).click();
+        $(Input.UPLOAD_AN_OPENAPI_FILE).shouldBe(visible).click();
```

Comment:
would be possible to use something like setValue() on radio element?

Reply:
Thanks for the idea, setValue() did not work but setSelected() does the job.